### PR TITLE
Homework 3 from Aurora

### PR DESCRIPTION
**Pull request by Aurora2500**

### CODE REVIEW

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845647)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845647&bid=16083669)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845647)

File: Src/BootCamp.Chapter/Program.cs [hunk 3]
```diff
@@ -9,6 +9,47 @@ class Program
...
+            return Console.ReadLine();
+        }
+
+        public static float PromptFloat(string prompt)
+        {
+            Console.Write(prompt);
+            return float.Parse(Console.ReadLine());
+        }
+
+        public static float CalculateBMI(float weight, float height)
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Program.cs]: Codacy Issue found: [Rename method 'CalculateBMI' to match pascal case naming rules, consider using 'CalculateBmi'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845647)

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+using System.Text;
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Codacy Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845647)

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845647&bid=16083669)

File: Src/BootCamp.Chapter/Program.cs [hunk 5]
```diff
@@ -1,14 +1,52 @@
...
+            float weight = PromptFloat("Weight: ");
+            float height = PromptFloat("Height: ");
+            
+            //print a sentence about user
+            Console.WriteLine($"{name} {surname} is {age} years old, their weight is {weight}Kg and their height is {height}cm");
+
+            //calculate and print the BMI of the user
+            float bmi = CalculateBmi(weight, height / 100);
+            Console.WriteLine($"They have a BMI of {bmi}");
+        }
```
Comment (ellmaster) [inline Src/BootCamp.Chapter/Program.cs]: That's a sneaky way to keep you code where it was. The task - as far as I understood it - wanted from you that you move the actual logic over to the Demo() function, not only a reference to the logic.

Comment (Aurora2500) [reply Src/BootCamp.Chapter/Program.cs]: You mean in the Descriptor() function?

Comment (ellmaster) [reply Src/BootCamp.Chapter/Program.cs]: Yeah.
Following tasks need to be done (as clarified by Kai moments ago)
> Move every functions except Main() to Lesson3.cs
> Change Program.X() to Lesson3.X() where needed.

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845647&bid=16083669)

File: Src/BootCamp.Chapter/Program.cs (hunk 5 above)
Comment (Aurora2500) [reply Src/BootCamp.Chapter/Program.cs]: I moved all the functions to Lesson3 and explicitly made two different float variables, one for the height in cm and one for the height in m.

File: Src/BootCamp.Chapter/Lesson3.cs [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            Descriptor();
+            Descriptor();
+        }
+
+        public static void Descriptor()
```
Comment (Almantask) [inline Src/BootCamp.Chapter/Lesson3.cs]: Is this really a good name for a method? I wouldn't know what it does without looking at the contents, please pick a better name.

Comment (Aurora2500) [reply Src/BootCamp.Chapter/Lesson3.cs]: Changed the method name to Interview(), it's as good of a name I can come up with.

**Comment by Almantask** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 3
           

Complexity increasing per file
==============================
- Src/BootCamp.Chapter/Lesson3.cs  1
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4845647&bid=16083669)

**Pull request closed without merging**